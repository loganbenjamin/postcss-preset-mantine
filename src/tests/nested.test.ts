const { testTransform } = require('./utils');

const baseInput = `
.a {
    .b {
        color: black;
    }
}
`;

// We don't have to test all functionality of postcss-nested, just that it's included
describe('nested', () => {
  it('should work', async () => {
    const res = await testTransform(baseInput);
    expect(res.css).toMatchSnapshot();
  });
});
