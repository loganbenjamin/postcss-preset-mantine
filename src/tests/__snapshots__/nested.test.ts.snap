// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`nested should work 1`] = `
"
.a .b {
        color: black;
    }
"
`;
